### DISCUSSION TITLE
Should we use Eclipse Corrosion to add Rust support to Eclipse for C/C++ Developers

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-cdt/cdt#949] [General]
There is the beginning of an interesting conversation in EPP - see https://github.com/eclipse-packaging/packages/issues/237

**Comment by ruspl-afed** [comment]
Eclipse Corrosion is heavily dependent on Eclipse CDT, so technically it shouldn't be a difficult task.
From the user's point of view, this can also be a plus, since the universes of C, C++ and Rust have many intersections.
